templates/bundles/SyliusShopBundle/Homepage/_carousel.html.twig

```diff
@@ -0,0 +1,19 @@
+<div class="carousel-wrapper">
```

Comment:
If we will manage to use the custom block template, then this file will not be needed

Reply:
this file is require at this moment